go/libraries/doltcore/diff/diff_summary.go

```diff
@@ -0,0 +1,338 @@
+// Copyright 2019 Liquidata, Inc.
```

Comment:
I copied a lot of this from noms - should I keep this file or make these changes [here](https://github.com/liquidata-inc/dolt/blob/master/go/store/diff/summary.go)?

Reply:
I believe this is correct.  If it's not there is a check that happens, and jenkins will fail your build.